src/sanitiser.js

```diff
@@ -25,31 +25,43 @@ const getDefaultUserAgent = function() {
   );
 };
 
-const sanitiseDefaultOptions = function(options) {
+const getTemplatesInfo = templates =>
+  templates.reduce((templatesHash, template) => {
+    templatesHash[template] = require(template).getInfo().version;
+    return templatesHash;
+  }, {});
+
+const sanitiseDefaultOptions = function(options, config) {
   if (_.isFunction(options)) {
     options = {};
```

Comment:
this should be part of the "copy" path as we are changing input if we do that,
but we can iterate later